lib/bloc/withdraw/withdraw_funds_bloc.dart

```diff
@@ -17,17 +17,78 @@ class WithdrawFundsBloc extends Cubit<WithdrawFundsState> {
     this._breezLib,
   ) : super(WithdrawFundsState.initial());
 
+  /* Reverse Swap */
+
+  Future<ReverseSwapInfo> sendOnchain({
```

Comment:
These functions should be in the reverse_swap bloc. The withdraw is related to the sweep when the user has on-chain balance (unexpected funds in breez mobile).

Reply:
Separeted ReverseSwap, Sweep & FeeOptions blocs 01ea0504972c88bc37c0e0185db32943e476d14a